File: Questions/Day-2/Cpp/Q3.cpp
#include <iostream>
using namespace std;

int main()
{
    cout << "Enter a: ";
    int a = 0;
    cin >> a;

    cout << "Enter b: ";
    int b = 0;
    cin >> b;

    cout << "Calculator\n1.Addition\n2.Subtraction\n3.Multiplication\n4.Division";

    cout << "Enter your choice: ";
    int choice = 0;
    cin >> choice;

    switch (choice)
    {
    case 1:
        cout << "\nAddition: " << (a + b);
        break;
    case 2:
        cout << "\nSubtraction: " << (a - b);
        break;
    case 3:
        cout << "\nMultiplication: " << (a * b);
        break;
    case 4:
        cout << "\nDivision: " << (a / b);
        break;
    default:
        cout << "Invalid Operation";
        break;
    }
}

File: Questions/Day 2/Java/Q3.java
import java.util.Scanner;

public class Q3 {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);

        System.out.print("Enter a: ");
        int a = in.nextInt();

        System.out.print("Enter b: ");
        int b = in.nextInt();

        System.out.println("Calculator\n1.Addition\n2.Subtraction\n3.Multiplication\n4.Division");

        System.out.print("Enter your choice: ");
        int choice = in.nextInt();

        switch (choice) {
            case 1:
                System.out.println("\nAddition: " + (a + b));
                break;
            case 2:
                System.out.println("\nSubtraction: " + (a - b));
                break;
            case 3:
                System.out.println("\nMultiplication: " + (a * b));
                break;
            case 4:
                System.out.println("\nDivision: " + (a / b));
                break;
            default:
                System.out.print("Invalid Operation");
                break;
        }
    }
}
